src/tests/loginUser.test.ts

```diff
@@ -1,25 +1,63 @@
 import axios from "axios";
 // import { loginUser } from "../services/userServices";
+describe('test for login procedure',()=> {
 
-const url = "http://localhost:8000/api/login"
+    const url = "http://localhost:8000/api/login"
```

Comment:
why is this static please make this a dynamic what if someone run this on different port